### PR TITLE
Tajikistan earthquake shakemaps

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
This PR includes shakemap layer served from geoserver as WMS/WFS request:

**Additional features:**
- Included getFeatureinfo props to get mmi value.

image

**Note:** Exposed population analysis is currently disabled. There is a bug within the table rendering step. Documented here: https://github.com/WFP-VAM/prism-frontend/issues/238

### CODE REVIEW

**Comment by JorgeMartinezG** [comment]
This branch has been updated and includes a fix for legend generation. Also, validated the exposed population analysis. cc @erwagasore @wadhwamatic 

image

**Review by erwagasore — APPROVED** [review]
Seems very straight forward. Ship it.

File: src/utils/analysis-utils.ts [hunk 1]
```diff
@@ -524,15 +524,17 @@ export function createLegendFromFeatureArray(
     (a, b) => a - b, // This function is required since JS assumes unicode values.
   );
 
-  const statsLen = stats.length;
+  const uniqueStats = [...new Set(stats)]; // Remove repeated elements.
```
Comment (ericboucher) [inline src/utils/analysis-utils.ts:527]: Can you explain a bit more when this is used and why it is important to remove the repeated elements?

*Technically speaking, the values below are not quintiles anymore if you pre-filter to remove duplicate values.

Comment (ericboucher) [reply src/utils/analysis-utils.ts:527]: Yes I understand, however:
- these are not quintiles anymore by definition (not a huge deal - my request was mostly to add comments in the code)

- this problem is a bit more complex than it seems, and this solution might not work exactly as you expect. I recommend adding some tests for this function with a few edge cases. Eg. what happens if there are less than 5 unique values? Then we would have 1,734,633 multiple times as well. -> Using unique values also risks adding its own edge cases.
- We could also envision using an equal value split between the min/max for example.

**Review by wadhwamatic — APPROVED** [review]
Nice work on this Jorge. I think the issue raised by Eric about equal interval vs quintile deserves a bit more thinking. For now, let's use the solution you've created, but I'll create another issue on this.

**Comment by wadhwamatic** [comment]
> Nice work on this Jorge. I think the issue raised by Eric about equal interval vs quintile deserves a bit more thinking. For now, let's use the solution you've created, but I'll create another issue on this.

See issue https://github.com/WFP-VAM/prism-frontend/issues/246
